### PR TITLE
MBS-7907: Warning about pending edits on a relationship is missing

### PR DESCRIPTION
**Pull request by mwiencek**
Trying to close old tickets that have been assigned to me since forever.

### CODE REVIEW

File: root/components/relationship-editor.tt [hunk 1]
```diff
@@ -10,11 +10,11 @@
 </div>
 
 <script type="text/html" id="template.relationship-dialog">
-  <!-- ko if: relationship().editsPending -->
+  <!-- ko if: $data.editing && $data.editing.editsPending -->
```
Comment (mwiencek) [inline root/components/relationship-editor.tt]: This fixes the bug. `$data` is the dialog, `editing` is a reference to the relationship being edited.
https://github.com/mwiencek/musicbrainz-server/blob/mbs-7907/root/static/scripts/relationship-editor/common/dialog.js#L570

File: root/components/relationship-editor.tt [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
     <p class="msg warning">
       [% l('Warning: This relationship has pending edits.
             {show|Click here} to view these edits and make sure they do not conflict with your own.',
-           { show => { 'data-bind' => 'attr: { href: relationship().openEdits() }', target => '_blank' } }) %]
+           { show => { 'data-bind' => 'attr: { href: $data.editing.openEdits() }', target => '_blank' } }) %]
```
Comment (mwiencek) [inline root/components/relationship-editor.tt]: This was broken too. :) `relationship()` is a clone of `editing`, as noted by chirlu in the ticket.

File: root/components/relationship-editor.tt (hunk 1 above)
Comment (gentlecat) [inline root/components/relationship-editor.tt]: Are these comments parsed by Template Toolkit or is it Knockout?

Comment (uklauer) [inline root/components/relationship-editor.tt]: The comments are Knockout, they call it a virtual element or something (it’s for grouping several tags without forcing us to introduce an unnecessary `<div>`, like React does it).

Comment (uklauer) [inline root/components/relationship-editor.tt]: http://knockoutjs.com/documentation/if-binding.html#note-using-if-without-a-container-element

Comment (gentlecat) [inline root/components/relationship-editor.tt]: Got it. Thanks.

**Comment by gentlecat** [comment]
LGTM :shipit:

**Pull request merged**